File: 02_ProgrammingFundamentals/_02_DataTypesAndVariables/_01_Lecture/_08_LowerOrUpper.java
package _02_ProgrammingFundamentals._02_DataTypesAndVariables._01_Lecture;

import java.util.Scanner;

public class _08_LowerOrUpper {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        String console = scan.nextLine();

        // Add variable
        char symbol = console.charAt(0);

        // Print result
        System.out.println(symbol >= 97 ? "lower-case" : "upper-case");

        scan.close();
    }
}

File: 24_C#Fundamentals/_02_DataTypesAndVariables/_01_Lecture/_08_LowerOrUpper/_08_LowerOrUpper.cs
using System;

public class _08_LowerOrUpper
{
    public static void Main(string[] args)
    {
        // Read input value
        string console = Console.ReadLine();

        // Add variable
        char symbol = console[0];

        // Print result
        Console.WriteLine(symbol >= 97 ? "lower-case" : "upper-case");
    }
}
